examples/asset/utilization/README.md

```diff
@@ -0,0 +1,34 @@
+Asset Utilization Example
+=====================
+
+This is a basic example console application demonstrating how to track asset utilization
+using SystemLink Asset Utilization APIs.
+
+Running the Example
+-------------------
+
+1. Download and extract the [repository source](https://github.com/ni/systemlink-client-docs/archive/master.zip).
+2. Install the [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core).
+3. Navigate to the example's directory and use the [`dotnet run` command](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-run?tabs=netcore21).
+
+To run the example, SystemLink Client needs to be installed. 
+
+About the Example
+-----------------
+
+This example tracks utilization for assets present in the current system. 
+The utilizations are stored localy and once a connection to the SystemLink Server is established,
```

Comment:
- Would it be accurate to describe the utilizations as "utilization logs" or "utilization data" instead? 
- "locally"

Reply:
I will go with utilization data. Thanks for the recommendations